test-util/src/main/java/io/zeebe/test/util/MsgPackUtil.java

```diff
@@ -120,11 +120,15 @@ public static void assertEqualityExcluding(
     assertThat(msgPackNode).isEqualTo(jsonNode);
   }
 
-  public static byte[] asMsgPack(String json) {
+  public static byte[] asMsgPackArray(String json) {
```

Comment:
Why the renaming to array?

Reply:
All the other `asMsgPack` methods return `DirectBuffer`. I needed a method with the signature `DirectBuffer <method-name>(String json)`, so I considered this renaming to be the best solution.

Reply:
Aaaaah now I get it. It is not meant to be json/msgpack array the array stands for the return type :D :see_no_evil: